controllers/users/authenticate-web.js

```diff
@@ -7,6 +7,7 @@ const {User} = require('../../databases/models');
 const BetterSocialCore = require('../../services/bettersocial');
 const {createRefreshToken} = require('../../services/jwt');
 const Getstream = require('../../vendor/getstream');
+const HumanIdService = require('../../vendor/humanid');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `HumanIdService` is a new addition and seems to be correctly imported. However, ensure that the path is correct and the exported module from the file at this path has a method called `verifyExchangeToken`.
```diff
+ const HumanIdService = require('../../vendor/humanid');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->